src/controllers/main/main.ts

```diff
@@ -1171,7 +1178,13 @@ export class MainController extends EventEmitter {
     // the time we get here. To prevent issues, we check one more time
     if (!this.signAccountOp) return
 
-    this.signAccountOp.update({ gasPrices: this.gasPrices[networkId] })
+    this.signAccountOp.update({
+      gasPrices: this.gasPrices[networkId],
+      bundlerGasPrices:
+        this.bundlerGasPrices && this.bundlerGasPrices[networkId]
```

Comment:
why? can't we just alwaysa initialize `this.bundlerGasPrices` and just use `this.bundlerGasPrices[networkId]`. Or simply `this.bundlerGasPrices?[networkId]`? but I vote for always initialize